### PR TITLE
Feature/variable

### PR DESCRIPTION
**Pull request by akira28**

### CODE REVIEW

**Comment by akira28** [comment]
Before you ask, ARG must be defined again because if you use arg before FROM it's not visible in the build

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,11 @@
-FROM nginx:1.16.0
+ARG VERSION=1.16.0
+FROM nginx:${VERSION}
```
Comment (rccrdpccl) [inline Dockerfile:2]: Will this work for docker hub automated builds?

**Review by bvis — APPROVED** [review]

**Review by rccrdpccl — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (bvis) [reply Dockerfile:2]: AFAIK yes

Comment (rccrdpccl) [reply Dockerfile:2]: My bad, I've misread the diff

**Pull request merged**